kotlin-dsl/src/main/kotlin/org/powerflows/kotlin/dsl/api.kt

```diff
@@ -0,0 +1,246 @@
+package org.powerflows.kotlin.dsl
+
+import org.powerflows.dmn.engine.DecisionEngine
+import org.powerflows.dmn.engine.configuration.DefaultDecisionEngineConfiguration
+import org.powerflows.dmn.engine.evaluator.expression.provider.binding.InstanceMethodBinding
+import org.powerflows.dmn.engine.evaluator.expression.provider.binding.MethodBinding
+import org.powerflows.dmn.engine.evaluator.expression.provider.binding.StaticMethodBinding
+import org.powerflows.dmn.engine.model.decision.Decision
+import org.powerflows.dmn.engine.model.decision.EvaluationMode
+import org.powerflows.dmn.engine.model.decision.HitPolicy
+import org.powerflows.dmn.engine.model.decision.expression.ExpressionType
+import org.powerflows.dmn.engine.model.decision.field.Input
+import org.powerflows.dmn.engine.model.decision.field.ValueType
+import org.powerflows.dmn.engine.model.decision.rule.Rule
+import org.powerflows.dmn.engine.model.decision.rule.entry.InputEntry
+import org.powerflows.dmn.engine.model.decision.rule.entry.OutputEntry
+import java.io.Serializable
+import java.lang.reflect.Method
+
+
+@DslMarker
```

Comment:
Formattig of the whole file is changing when I use Ctrl+Alt+L in IDEA. Do I need some special settings, or just you need to format before a merge of this PR?